backend/src/main/java/hu/flowacademy/meteo/controller/StationController.java

```diff
@@ -0,0 +1,24 @@
+package hu.flowacademy.meteo.controller;
+
+import hu.flowacademy.meteo.dto.StationDto;
+import hu.flowacademy.meteo.service.StationService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequiredArgsConstructor
+public class StationController {
+
+    private final StationService stationService;
+
+    @CrossOrigin("*")
+    @GetMapping("api/coordinates")
```

Comment:
Találj jobb nevet a metódusnak, a findAll egy service-ben sem megy már el. Lehetne pl listStations (ugyanígy a service osztályban is). gondom van a mappinggal is, mert ezt a backend szempontjából kell értelmezni, nem onnan, hogy mire fogja használni a UI. A coordinates kulcsszót csak gyengén tudom kapcsolni (a station-nak van koordinátája). Az "api/stations" egy az igazságnak jobban megfelel

Reply:
@akesztyus @Godfrid Másik, nem akarok belekontárkodni itt a backendbe, de rengeteg helyen látom, hogy a mapping-be bekerül a /api prefix mindenhol (másik projekten is). Ennél egy fokkal jobb lenne, a controllerek elején egy @RequestMapping vagy még ügyesebb lenne egy globális beállítás, például a context-path. Mit gondoltok?

Reply:
@VassTamas87 Attila észrevétele továbbra is áll, lehetne stations inkább.